version.props

```diff
@@ -1 +1 @@
-3.2.7
\ No newline at end of file
+3.2.8
```

Comment:
Actually, wait, shouldn't we be on 3.2.9 now? We never should have released 3.2.8 at all.

Reply:
Should we release a 3.2.8 on NuGet at all then? Or re-do a release for both npm and NuGet?

Reply:
Chatted in person. I think we should just "reset" and go to 3.2.9 at this point.